warehouse/utils/project.py

```diff
@@ -65,6 +66,32 @@ def confirm_project(
         )
 
 
+def prohibit_and_remove_project(
+    project: Project | str, request, comment: str, flash: bool = True
+):
+    """
+    View helper to prohibit and remove a project.
+    """
+    # TODO: See if we can constrain `project` to be a `Project` only.
```

Comment:
I intend to come back to this, and probably take a `str` only, since we don't actively use the `Project` object other than the name. It's resolved to a `Project` later on before calling `remove_project()`.